recipes/pangeo-notebook/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: pangeo-notebook
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask-gateway >=0.6
+    - dask-labextension >=1.1
+    - jupyter-server-proxy >=1.2
+    - jupyterhub >=1.1
+    - jupyterlab >=1.2
+    - nbgitpuller >=0.8
+    - pangeo-dask >=0.0.2
+
+test:
+  commands:
+    - jupyter -h
```

Comment:
Both tests are dummy metapackage tests. Don't worry about them.

Reply:
ok! thanks so much for doing this.